mobile/lib/widgets/custom_widgets.dart

```diff
@@ -42,25 +42,45 @@ class HealthTipContainer extends StatelessWidget {
           Radius.circular(16.0),
         ),
       ),
-      child: Row(
-        children: [
-          Container(
+                child: Row(
+                  children: [
+                    Container(
             constraints: const BoxConstraints(
               maxWidth: 83,
               maxHeight: 112,
               minWidth: 83,
               minHeight: 112,
             ),
-            decoration: BoxDecoration(
-              borderRadius: BorderRadius.circular(8.0),
-              image: DecorationImage(
-                fit: BoxFit.cover,
-                image: CachedNetworkImageProvider(
-                  healthTip.image,
+            child: CachedNetworkImage(
+              imageUrl: healthTip.image,
```

Comment:
Thanks @katendejericho5. Let us also add this functionality to the home screen KYA images.